src/core/lombok/bytecode/PoolConstantsApp.java

```diff
@@ -66,7 +66,7 @@
 		CmdReader<CmdArgs> reader = CmdReader.of(CmdArgs.class);
 		CmdArgs args;
 		try {
-			args = reader.make(raw.toArray(new String[0]));
+			args = reader.make(raw.toArray(new String[raw.size()]));
```

Comment:
Hmm... Yes, you're completely right! I thought that score was measured in "ms", but it was "ops/ms".

I've just re-read this articles carefully and situation with `toArray` is really interesting. Thank you for information! This PR could be closed, sorry for taking your time.